### ISSUE TITLE
Feature Request: Map Types-View Rules as config

### ISSUE THREAD

**Issue by GloriousPtr** [EpicGamesExt/raddebugger#118]
It may sound a bit like natvis but it can be an alternative way to do it. Close the ticket if you feel like it's not a good way to do things.

A mapping of types with view rules so that watch variables should have that view rule for specific types applied by default.
For instance:
| Type | Rule |
|---|---|
| eastl::vector | array:($.mpBegin, $.mpEnd-$.mpBegin) | 

$- wildcard for variable iteself and view rule array is modified to take in start pointer and element count.

Edit 1: This map can be saved to the profile i guess?

**Comment by ryanfleury** [comment]
Yep this is already on my [TODO list](https://github.com/EpicGames/raddebugger/blob/master/src/raddbg/raddbg.h#L378) actually.

**Comment by ryanfleury** [comment]
Completed in 0.9.16.
